package.json

```
@@ -47,9 +50,30 @@
     "vue2-daterange-picker": "^0.6.8",
     "vuedraggable": "^2.24.3",
-    "wp-readme-to-markdown": "^1.0.1"
+    "wp-readme-to-markdown": "^1.0.1",
+    "tailwind-merge": "^2.6.0"
   },
   "dependencies": {
-    "@wordpress/i18n": "^5.8.0"
+    "@dnd-kit/core": "^6.3.1",
+    "@dnd-kit/sortable": "^10.0.0",
+    "@getdokan/dokan-ui": "^1.0.18",
+    "@wordpress/api-fetch": "^7.14.0",
```

Biome (1.9.4)

[error] The key @wordpress/api-fetch was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)